swagger/swagger.yaml

```diff
@@ -49,8 +49,8 @@ paths:
           default: false
       - name: max_scenarios
         in: query
-        description: "the maximum number of scenarios requested, supported only in\
-          \ `test` sessions"
+        description: "the maximum number of scenarios requested, not supported in\
```

Comment:
same question as in server, reading the before and after suggests this wouldnt be supported at all?

Reply:
As in server, `max_scenarios` is supported in TA1 (`soartech` & `adept`) sessions, but not `eval` sessions.  Basically I removed `test`, but made sure that `max_scenarios` works in `soartech` & `adept`.